code/modules/projectiles/guns/ballistic.dm

```diff
@@ -286,6 +286,10 @@
 	update_appearance()
 	update_item_action_buttons()
 
+#define CASING_CATCH_NO_ATTEMPT	0
+#define CASING_CATCH_FAILED	1
```

Comment:
```suggestion
#define CASING_CATCH_FAILED 1
```

no need for cool spacing here especially when it's mixing tabs and spaces

Reply:
i would like to blame my vscode instance for this